### DISCUSSION TITLE
`syncInput` sends a `null` name

### DISCUSSION THREAD

**Discussion by ryangjchandler** [livewire/livewire#3716] [7. Bugs (archived)] [answered]
Ran into a weird error on the latest version. An input with `wire:model` (not lazy or deferred) correctly triggers a `syncInput` call but the `name` property is `null`.

<img width="304" alt="CleanShot 2021-09-14 at 13 39 26@2x" src="https://user-images.githubusercontent.com/41837763/133259273-3a90e15e-e888-4a0d-aad2-4d4a6b2882d1.png">

This triggers the following error:

<img width="571" alt="CleanShot 2021-09-14 at 13 41 20@2x" src="https://user-images.githubusercontent.com/41837763/133259318-1e9e1781-4092-4659-90ef-62d11d37cddc.png">

**Comment by ryangjchandler** [comment, marked as answer]
Not an issue with Livewire - was an issue caused by integration with a third-party.

**Reply by rodrigoross** [reply]
Actually I'm having the same issue, how did you solved it because I'm not using any third party in the current component.
